autoextract_spiders/spiders/autoextract_article.py

```diff
@@ -109,7 +125,8 @@ def parse_feed(self, response: XmlResponse):
             yield Request(url,
                           meta={'source_url': source_url, 'feed_url': feed_url},
                           callback=self.parse_page,
-                          errback=self.errback_page)
+                          errback=self.errback_page,
+                          dont_filter=True)
```

Comment:
At this point I believe we would need the switch, because we don't want to parse the Article URL coming from the feed everytime.
The idea is to extract the article only once and remember the URL, and potentially never extract the article again - unless for devel, or debug.